Sources/App/Models/Karma/KarmaSlackHistory.swift

```diff
@@ -6,29 +6,29 @@
 //
 
 import Foundation
-import FluentSQLite
+import FluentPostgreSQL
 import Vapor
 
 /// Captures a karma interaction from Slack
-final class KarmaSlackHistory: SQLiteModel {
+final class KarmaSlackHistory: PostgreSQLModel {
     var id: Int?
 
     var karmaCount: Int
-    var fromUser: String
     var karmaReceiver: String
-    var channel: String
+    var karmaSender: String
+    var inChannel: String
 
     init(id: Int? = nil,
          karmaCount: Int,
-         fromUser: String,
          karmaReceiver: String,
-         channel: String) {
+         karmaSender: String,
```

Comment:
These could be `sender` and `recipient`, just thinking, not a hill I'm gonna die on

Reply:
I like that they all were prefaced with `karma`